CK3toEU4/Source/EU4World/Country/Country.cpp

```diff
@@ -1,6 +1,9 @@
 #include "Country.h"
```

Comment:
In the long term there should be tests for all of this. The java builder pattern (not to be confused with the builder pattern. Naming is hard.) I've been using in Vic2ToHoI4 can be used to set up CK3 and mapper objects for test setup.